x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_settings/conversation_settings.tsx

```diff
@@ -382,67 +120,18 @@ export const ConversationSettings: React.FC<ConversationSettingsProps> = React.m
           onConversationSelectionChange={onConversationSelectionChange}
         />
 
-        <EuiFormRow
-          data-test-subj="prompt-field"
-          display="rowCompressed"
-          fullWidth
-          label={i18n.SETTINGS_PROMPT_TITLE}
-          helpText={i18n.SETTINGS_PROMPT_HELP_TEXT_TITLE}
-        >
-          <SelectSystemPrompt
-            allSystemPrompts={allSystemPrompts}
-            compressed
-            conversation={selectedConversation}
-            isEditing={true}
-            isDisabled={isDisabled}
-            onSystemPromptSelectionChange={handleOnSystemPromptSelectionChange}
-            selectedPrompt={selectedSystemPrompt}
-            showTitles={true}
-            isSettingsModalVisible={true}
-            setIsSettingsModalVisible={noop} // noop, already in settings
-            isFlyoutMode={isFlyoutMode}
-          />
-        </EuiFormRow>
-
-        <EuiFormRow
-          data-test-subj="connector-field"
-          display="rowCompressed"
-          label={i18n.CONNECTOR_TITLE}
-          helpText={
-            <EuiLink
-              href={`${http.basePath.get()}/app/management/insightsAndAlerting/triggersActionsConnectors/connectors`}
-              target="_blank"
-              external
-            >
-              <FormattedMessage
-                id="xpack.elasticAssistant.assistant.settings.connectorHelpTextTitle"
-                defaultMessage="Kibana Connector to make requests with"
-              />
-            </EuiLink>
-          }
-        >
-          <ConnectorSelector
-            isDisabled={isDisabled}
-            onConnectorSelectionChange={handleOnConnectorSelectionChange}
-            selectedConnectorId={selectedConnector?.id}
-            isFlyoutMode={isFlyoutMode}
-          />
-        </EuiFormRow>
+        <ConversationSettingsEditor
+          allSystemPrompts={allSystemPrompts}
+          conversationSettings={conversationSettings}
+          conversationsSettingsBulkActions={conversationsSettingsBulkActions}
+          http={http}
+          isDisabled={isDisabled}
+          isFlyoutMode={isFlyoutMode}
+          selectedConversation={selectedConversationWithApiConfig}
```

Comment:
This is to fix conversation not updated when changing a prompt:


https://github.com/elastic/kibana/assets/6295984/c32c0c21-200f-4773-bedf-3d82f391c383